bitnami/consul/templates/statefulset.yaml

```diff
@@ -67,28 +81,35 @@ spec:
             - name: data
               mountPath: /bitnami/consul
         {{- end }}
+        {{- if .Values.initContainers }}
+        {{- include "common.tplvalues.render" (dict "value" .Values.initContainers "context" $) | nindent 8 }}
+        {{- end }}
+      {{- end }}
       containers:
         - name: consul
```

Comment:
I'm missing the custom `command`/`args` option that can be specified in values.yaml. I see it was added to both the README and values files though.

Please make sure there are no unused values.